### PR TITLE
feat(tmpl): support glusterfs

### PR DESCRIPTION
**Pull request by zjj2wry**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by zjj2wry** [comment]
/assign @zoumo 

已经测试过了，ptal

**Comment by zjj2wry** [comment]
@li-ang clever 这边要夸分区、夸分区共享存储，这里的 glusterfs 是 pod 直接使用 glusterfs 的 volume，不走 pv、pvc 那一套

File: templates/1.0.0/templates/_volumes.tpl [hunk 1]
```diff
@@ -48,6 +48,12 @@
       mode: {{ .mode }}
     {{- end -}}
 {{ end -}}
+{{- if eq .type "Glusterfs" }}
+  glusterfs:
+    endpoints: {{ .source.endpoints | quote }}
+    path: {{ .source.path }}
```
Comment (zoumo) [inline templates/1.0.0/templates/_volumes.tpl]: {{ .source.path | quote }}

**Comment by zjj2wry** [comment]
//cherrypick release-1.1

**Pull request merged**